clicommand/redactor_add.go

```diff
@@ -57,25 +57,27 @@ redacted from subsequent logs. Secrets fetched with the builtin
 ′secret get′ command do not require the use of this command, they will
 be redacted automatically.
 
-Example:
+Examples:
 
 To redact the verbatim contents of the file 'id_ed25519' from future logs:
 
-    $ buildkite-agent redactor add id_ed25519
+        $ buildkite-agent redactor add id_ed25519
```

Comment:
Why is the indentation changing?

This example uses 8 spaces, the next example uses 2 tabs. My personal preference would be 1 tab for all of them, but whatever we do, can we please ensure the indented lines are consistently indented?